clients/vscode-hlasmplugin/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## ****Unreleased****
 
+#### Added
+- Highlight lines in listings that originate from macros and copybooks
```

Comment:
Not sure - did you want to highlight complete lines are just the stmt number?

Reply:
Just stmt numbers, in order to convey a message about the origin of the whole line, so I think the description is ok.